spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts' do
```

Comment:
- [ ] `index` and `show` actions for `User` and `Post` controllers should be tested like below:

```
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```